tsconfig.test.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "composite": false,
     "emitDeclarationOnly": false,
+    "isolatedModules": false,
```

Comment:
This looks better than empty `export {}`. Or? There will be more tests without `imports`, so repeating `export {}` seems redundant.

Reply:
we could just add `import {expect} from '@jest/globals'`? or can't we do that due to cycles?